packages/contracts-bedrock/test/kontrol/proofs/L1CrossDomainMessenger.k.sol

```diff
@@ -19,25 +19,18 @@ contract L1CrossDomainMessengerKontrol is DeploymentSummary, KontrolUtils {
         superchainConfig = SuperchainConfig(superchainConfigProxyAddress);
     }
 
-    /// TODO: Replace struct parameters and workarounds with the appropriate
-    /// types once Kontrol supports symbolic `bytes` and `bytes[]`
-    /// Tracking issue: https://github.com/runtimeverification/kontrol/issues/272
     function prove_relayMessage_paused(
         uint256 _nonce,
         address _sender,
         address _target,
         uint256 _value,
-        uint256 _gas
+        uint256 _gas,
+        bytes calldata _message
```

Comment:
So this has a max length of 2^63 because of `maxBytesLength` in the lemmas file? Does that affect performance at all? Realistically we're bound by gas costs here—if everything else was free, we'd have 30M gas to spend on all zero bytes of calldata at 4 gas per byte, giving us 30M/4 = 7.5M bytes in length, meaning we could safely shrink that max length

Reply:
AFAIK the max size of the `bytes` parameter doesn't affect the performance. I'll shrink that max length and see if we win some performance. What causes a bit of a slow down is having a symbolic length (capped at 2⁶³) vs a concrete length, although by what I've observed it's just a couple of minutes on my machine

Reply:
Following up here too to see if we should shrink the max length

Reply:
After discussing internally, shrinking the max length of the `bytes` parameters would not yield any performance benefits, so there is no harm in having the most general (although, realistically, overkill) bound. Note that the `maxBytesLength` is used only for the `copy_memory_to_memory` summary, which is relevant for `bytes[]` parameters. The `2^63` length bound is only applied to symbolic `bytes` parameters that are not elements of a `bytes[]`